src/components/MultiHopTrade/MultiHopTrade.tsx

```diff
@@ -123,31 +74,33 @@ export const MultiHopTrade = memo(
       return ''
     }, [defaultSellAssetId, sellChainId, sellAssetSubId])
 
-    const routeSellAsset = useAppSelector(state => selectAssetById(state, sellAssetId))
-    const routeBuyAsset = useAppSelector(state => selectAssetById(state, buyAssetId))
+    const sellAsset = useAppSelector(state => selectAssetById(state, sellAssetId))
+    const buyAsset = useAppSelector(state => selectAssetById(state, buyAssetId))
```

Comment:
Renamed for the sake of sanity, this isn't necessarily *route* buy/sell asset but may be a default, see `sellAssetId`/`buyAssetId` definition above